src/quo2/foundations/colors.cljs

```diff
@@ -44,7 +44,7 @@
 
 ;;Solid
 
-
+(def neutral-2-5 "#FAFBFC")
```

Comment:
-5 is opacity? generally we mark that as `-opa-5` if so

Reply:
@J-Son89 `2.5` opacity

Reply:
Can we ask the design team to rename it? or we can add a comment next to that color definition.

Reply:
Maybe we just use some other notation? -2_5? Or -2-dot-5? 🤷‍♂️

Reply:
@J-Son89 fixed it!